rasa/shared/core/training_data/story_reader/story_reader.py

```diff
@@ -55,7 +55,19 @@ def __init__(
         self._is_used_for_training = is_used_for_training
         self._is_parsing_conditions = False
 
-    def read_from_file(self, filename: Text) -> List[StoryStep]:
+    def read_from_file(
+        self, filename: Text, skip_validation: bool = False
+    ) -> List[StoryStep]:
+        """Reads stories or rules from file.
+
+        Args:
+            filename: Path to the story/rule file.
+            skip_validation: `True` if the file was already validated
```

Comment:
I'd change the description and just mention that this flag will skip the validation step. I don't think that users of this API will benefit from mentioning the database in this context 🤔